example-sdl.json

```diff
@@ -1,7 +1,8 @@
 {
   "source": "./package.json",
   "scripts": {
-    "run": "esy '@example-sdl' x TestSdl"
+    "run": "esy '@example-sdl' x TestSdl",
+    "run-bytecode": "dune exec example-sdl/TestSdl.bc"
```

Comment:
Might need to test this on Windows - I don't think this command will work as-is. Might need to be `esy b dune exec example-sdl/TestSdl.bc`. Is there a reason we can't run `esy '@example-sdl' x TestSdl.bc.exe`?

Reply:
I fixed this, sorry about that. I had to add an `install` stanza in the `dune` file to bring the `TestSdl.bc` file into the `bin` directory. Fixed now!

Reply:
Thanks Zach!